pkg/services/config.go

```diff
@@ -58,10 +57,8 @@ type Config struct {
 	DevMode                    bool     `usage:"Enable development mode" default:"false" name:"dev-mode" env:"OBOT_DEV_MODE"`
 	DevUIPort                  int      `usage:"The port on localhost running the dev instance of the UI" default:"5173"`
 	AllowedOrigin              string   `usage:"Allowed origin for CORS"`
-	ToolRegistry               string   `usage:"The tool reference for the tool registry" default:"github.com/obot-platform/tools"`
+	ToolRegistries             []string `usage:"The remote tool references to the set of tool registries to use" default:"github.com/obot-platform/tools" split:"true"`
 	WorkspaceProviderType      string   `usage:"The type of workspace provider to use for non-knowledge workspaces" default:"directory" env:"OBOT_WORKSPACE_PROVIDER_TYPE"`
-	WorkspaceTool              string   `usage:"The tool reference for the workspace provider" default:"github.com/gptscript-ai/workspace-provider"`
-	DatasetsTool               string   `usage:"The tool reference for the dataset provider" default:"github.com/gptscript-ai/datasets"`
```

Comment:
These are being removed because they can now be remapped directly with the `GPTSCRIPT_TOOL_REMAP` environment variable; e.g. `export GPTSCRIPT_TOOL_REMAP=github.com/gptscript-ai/workspace-provider=../tools/workspace-provider`